sass/styles.scss

```diff
@@ -247,6 +247,41 @@ Navigation
   transition: all 300ms ease-in-out;
 }
 
+/* ==========================================================================
+Modal Tabs
+========================================================================== */
+.attractions {
+  display: none;
+}
+
+.bar {
+  width: 100%;
+  overflow: hidden;
+}
+
+.bar .bar-item {
+  padding: 8px 16px;
+  float: left;
+  width: auto;
+  border: none;
+  display: block;
+  outline: 0;
+  background: $light-gray;
+  
+
+  &:hover {
+    background: $mid-gray;
+    color: white;
+  }
+}
+
+// the selected tab in the tab bar
+.bar .selected {
+  background: $mid-gray !important;
```

Comment:
Using !important isn't great. Might want to change hierarchy instead later.

Reply:
It is c/p'ed from a W3 example that I don't fully understand. Feel free to make it follow best practices

Reply:
👍 will address in a future commit!